### ISSUE TITLE
Cannot edit saved files if libxerces2-java is installed

### ISSUE THREAD

**Issue by ktgw0316** [Aries85/LightZone#169]
LightZone can't open JPEG-LZN files if libxerces2-java is installed on Ubuntu 15.10. Metadata in Info panel are empty for the lzn files. LZ gives a lot of `XMLException`s like following when executed in a terminal.

```
com.lightcrafts.utils.xml.XMLException: Expected child "Controls" under "HTML"
	at com.lightcrafts.utils.xml.XmlNode.getChild(XmlNode.java:142)
	at com.lightcrafts.ui.templates.TemplateTreeNode.<init>(TemplateTreeNode.java:37)
	at com.lightcrafts.ui.templates.TemplateNamespaceTreeNode.<init>(TemplateNamespaceTreeNode.java:30)
...
```

**Comment by ktgw0316** [comment]
`sudo mv /usr/share/java/xercesImpl-2.11.0.jar{,.bak}` fixes the problem.

**Comment by ikilledmypc** [comment]
I do not have this jar file on my computer (Fedora 23) yet I still have the same issue when I try to click the edit button on one of my Nikon .NEF files. It also displays an error box with: "this file has no version number" 
[trace.txt](https://github.com/Aries85/LightZone/files/265116/trace.txt)

**Comment by ktgw0316** [comment]
I think this issue was fixed with b70a616, a8d6591, and c53272.
